src/Agent/NewRelic/Agent/Core/Segments/MessageBrokerSerializationSegmentData.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using NewRelic.Agent.Core.Aggregators;
+using NewRelic.Agent.Core.Metrics;
+using NewRelic.Agent.Core.Time;
+using static NewRelic.Agent.Core.WireModels.MetricWireModel;
+using NewRelic.Agent.Configuration;
+
+namespace NewRelic.Agent.Core.Segments
+{
+    public class MessageBrokerSerializationSegmentData : AbstractSegmentData
```

Comment:
If we end up removing the two methods in `Transaction`, we can remove this class as well.